comet-java-client/src/main/java/ml/comet/experiment/OnlineExperiment.java

```diff
@@ -180,4 +180,65 @@ void logRemoteAsset(URI uri, String logicalPath, boolean overwrite,
      * @throws ArtifactException if operation failed.
      */
     CompletableFuture<LoggedArtifact> logArtifact(Artifact artifact) throws ArtifactException;
+
+    /**
+     * Logs the model data from specified folder.
+     *
+     * @param modelName   the name of the model.
+     * @param folder      the folder with model's data files.
+     * @param logFilePath if {@code true} than prefix file path of each data file with folder name.
+     * @param metadata    the metadata to be associated with model. The dictionary values must be JSON compatible.
```

Comment:
The metadata in this cases will be associated with the experiment's model asset.

Reply:
Fixed